src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -51,6 +52,10 @@
 public final class CollectSequencingArtifactMetrics extends SinglePassSamProgram {
     static final String USAGE = "Produces metrics to quantify single-base sequencing artifacts from a SAM/BAM file";
 
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "File to write the output to.")
+    public File OUTPUT;
```

Comment:
lowercase. 

btw, why was the output field moved from `SinglePassSamProgram` down to subclasses?